circle.yml

```diff
@@ -3,7 +3,8 @@ machine:
     version: 2.7.10
 dependencies:
   override:
-    - "pip install -U pip wheel setuptools"
+    - "pip install -U pip wheel"
+    - "pip install setuptools==24.3.1"
```

Comment:
Nit: `yml` files support comments; can you please add a link here to https://github.com/open-craft/problem-builder/commit/51277a34fb426724616618c1afdb893ab2de4c6b in case people are wondering why this is pinned and don't do a git blame?